app/models/data.js

```diff
@@ -52,4 +52,26 @@ Data.prototype.delete = function (key, callback) {
   });
 };
 
+Data.prototype.deleteOlderThanDate = function (date, callback) {
+  this.db.find({}, (err, docs) => {
+    if (err) {
+      return callback(err);
+    }
+
+    if (docs.length > 0) {
+      docs.forEach((item)=>{
+        let updatedAtDate = new Date(item.updatedAt);
+        if (updatedAtDate.getTime() < date) {
```

Comment:
@tejohnso yes, I was surprised but I think it does not filter by the updateAt because it is an internal field add by the database it self. If we had timestamp field that we added ourselves that the filtering would work. We could add this timestamp field now but I am worried that we would not cleanup old data that does not have the timestamp on it. What do you think? cheers